sphinxcontrib/needs/roles/need_outgoing.py

```diff
@@ -74,6 +77,19 @@ def process_need_outgoing(app, doctree, fromdocname):
                     else:
                         new_node_ref = nodes.reference(target_need["id"], target_need["id"])
                         new_node_ref["refuri"] = target_need["external_url"]
+
```

Comment:
okay, we have the same code on different location.
That's hard to maintain, if we detect an error in it.
Then we need to fix it at >4 locations.

Any chance to create a common function for this? maybe in ``util.py``